src/ssl.cxx

```diff
@@ -31,8 +32,15 @@
 #ifdef HAVE_LIBSSL
 static std::unique_ptr<SSL_CTX, std::function<void(SSL_CTX*)>> ssl;
 
+#if OPENSSL_VERSION_NUMBER < 0x10000000L
```

Comment:
FYI, I've had to change this to `0x10100000L` since it apparently does not work with 1.0.2n.